js/main.js

```diff
@@ -0,0 +1,29 @@
+// Решение подсмотрел здесь https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+function getRandomInteger(min, max) {
+  if (min > max || min === max) {
```

Comment:
Это условие можно сократить
```suggestion
  if (min >= max) {
```